src/support.js

```diff
@@ -121,3 +120,7 @@ exports.read = function read (reader) {
     lp.decodeFromReader(reader, { fixed: true, bytes: 4 }, (err, res) => err ? reject(err) : resolve(res))
   })
 }
+
+exports.wrap = (duplex) => {
```

Comment:
What's this for?

Reply:
:sweat_smile: some leftovers.

Currently pushed some broken code, just to iterate a bit how to do the streaming.

Unsure where I should wrap the stream: Before passing it to handshake and then using that object (as I did now) or using the support lib and a handshake stream.